src/cli/add.rs

```diff
@@ -55,13 +56,10 @@ pub struct Args {
     /// Don't install the package to the environment, only add the package to the lock-file.
     #[arg(long)]
     pub no_install: bool,
-}
 
-#[derive(Debug, Copy, Clone)]
-pub enum SpecType {
-    Host,
-    Build,
-    Run,
+    /// The platform(s) for which the dependency should be added
+    #[arg(long)]
+    pub platforms: Option<Vec<Platform>>,
```

Comment:
The CLI doesn't allow for multiple platforms as they seem to get used by the add command. I think that's fine though so I would just rename to `platform` as will for the documentation and the docstrings.  Don't think this has to be a vec.